templates/plugin/openscap.yml.erb

```diff
@@ -0,0 +1,9 @@
+---
+:enabled: <%= @enabled %>
+
+# Log file for the forwarding script.
+#:abrt_send_log_file: /var/log/foreman-proxy/openscap-send.log
```

Comment:
oops I missed abrt when I copied example template, seems it was already [fixed](https://github.com/OpenSCAP/foreman-proxy_openscap/commit/72602bb4942ac3092f1f831dda594aef472429aa) :-) 

I'll add parametrisation for both options